### DISCUSSION TITLE
Swiper v7 CSS being served from old CDN without .swiper-container fallback

### DISCUSSION THREAD

**Discussion by TCHref** [nolimits4web/swiper#4882] [Q&A]
Currently all sites that were using the Swiper CDN to serve files have broken carousels that overflow. It looks like there's a fallback for .swiper-container in the Javascript, but the .swiper-container CSS class has no styles at all.

Is this intended behaviour? Just want to double check if this was intentional before I run through and replace the markup for any project affected, thanks!

**Comment by nolimits4web** [comment]
https://swiperjs.com/migration-guide
